docs/reference/ml/delayed-data-detection.asciidoc

```diff
@@ -0,0 +1,43 @@
+[role="xpack"]
+[[ml-delayed-data-detection]]
+=== Working with delayed data
+
+In short, delayed data are any documents that have been indexed *AFTER* a Datafeed has gathered data
+from the index at that point in time.
+
+When creating a {ref}/ml-datafeed-resource.html[Datafeed Configuration], one may supply the option
+`query_delay`. This allows the Datafeed to wait for some time past real-time to allow any "late" data to be fully index
+before trying to gather it. However, if the setting is set too low, the Datafeed may query for data before it has been
+indexed, and consequently missing that document. Conversely, if it is set too high, analysis drifts farther away from
+real-time. The balance that is struck depends upon each use case and the environmental factors of the cluster.
+
+==== Why worry about delayed data?
+
+This is a particular prescient question. Certain types of detectors are not really effected by randomly missing data
+due to ingest delays. Statistically, it all comes out ok in the end as the missing data is distributed randomly. An
```

Comment:
It is unclear what part of the sentence "due to" applies to. Maybe makes more sense to say something like:
If there is data missing randomly (for example, due to ingestion delays), certain types of detectors are unaffected.